daphne/src/roles.rs

```diff
@@ -231,6 +237,24 @@ pub trait DapLeader<S>: DapAuthorizedSender<S> + DapAggregator<S> {
             collect_req.agg_param
         );
 
```

Comment:
`http_post_aggregate_share` should implement this check as well, right?

Reply:
Oops, you're right.